### PR TITLE
Fix typo in example IP requirements

### PR DESCRIPTION
**Pull request by Li-Yanzhi**

### CODE REVIEW

**Comment by ktoliver** [comment]
@mkostersitz Could you review the proposed update? Note that the same sentence structure is used in line 160:

image

File: azure-stack/aks-hci/concepts-node-networking.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ Based on the table above, she will have to reserve:
 
 - 3 IP addresses for the AKS host (one IP for control plane node and two IPs for running update operations)
 - 3 IP addresses for the control plane nodes in cluster A (one IP per control plane node)
-- 5 IP addresses for the worker nodes in cluster A (one IP per control plane node)
+- 5 IP addresses for the worker nodes in cluster A (one IP per worker node)
```
Comment (mkostersitz) [inline azure-stack/aks-hci/concepts-node-networking.md:157]: This is a valid change.

File: azure-stack/aks-hci/concepts-node-networking.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ Based on the table above, she will have to reserve:
...
 - 6 IP addresses additionally for cluster A (five IPs for running update operations and 1 IP for load balancer)
 - 1 IP addresses for the control plane nodes in cluster B (one IP per control plane node)
 - 3 IP addresses for the worker nodes in cluster B (one IP per control plane node)
```
Comment (mkostersitz) [inline azure-stack/aks-hci/concepts-node-networking.md:160]: We should update this as well as it is the same copy-paste error

**Review by mkostersitz — APPROVED** [review]
Please make the updates to both lines

**Comment by PRMerger9** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: azure-stack/aks-hci/concepts-node-networking.md (hunk 2 above)
Comment (ktoliver) [reply azure-stack/aks-hci/concepts-node-networking.md]: I applied the revision.

**Comment by ktoliver** [comment]
Fixed in both instances. Thanks for confirming @mkostersitz. I'll merge the PR. 

Thanks, all.

**Pull request merged**